docs/self-managed/zeebe-deployment/exporters/camunda-exporter.md

```diff
@@ -0,0 +1,185 @@
+---
+id: camunda-exporter
+title: "Camunda exporter"
+sidebar_label: "Camunda Exporter"
+description: "The common camunda exporter exporting zeebe records to harmonized web-apps indices ."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The Camunda Exporter replaces the old Elasticsearch and Opensearch exporters. Previously, these exporters mainly
```

Comment:
I'm doubtful myself if `replaces the old Elasticsearch and Opensearch exporters` is a correct statement.

Reply:
I removed this statement and re-phrased the paragraph